CONTRIBUTING.md

```diff
@@ -145,7 +145,7 @@ If you modify or add new API endpoints, update the API documentation accordingly
 
 ```bash
 $ uv sync --extra dev
-$ ./docs/openapi_generator/run_openapi_generator.sh
+$ uv run ./docs/openapi_generator/run_openapi_generator.sh
```

Comment:
I had no clue you could run arbitrary scripts via `uv run`

Reply:
uv is a gem!